src/components/forms/Search/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { History, Location } from 'history';
+import { trimStart } from 'lodash';
+import querystring from 'querystring';
+import React, { useEffect, useState } from 'react';
+import { Button, Form, FormGroup, Input } from 'reactstrap';
+import { SEARCH } from '../../../configs/lang';
+
+/**
+ * Interface for handleSearchChange event handler
+ */
+export type Change = (event: React.ChangeEvent<HTMLInputElement>) => void;
+
+/**
+ * Interface for handleSubmit event handler
+ */
+export type Submit = (event: React.FormEvent<HTMLFormElement>) => void;
+
+/**
+ * Interface for SearchForm props
+ */
+export interface SearchFormProps {
+  history: History;
+  location: Location;
+  placeholder?: string;
+}
+
+/** Search Form component */
+export const SearchForm = (props: SearchFormProps) => {
```

Comment:
I think that eventually we want this to be a [formik form](https://jaredpalmer.com/formik/).  But I am fine with us doing that in the future.  cc: @p-netm

Reply:
@kelvin-muchiri will you create a ticket for this before merging? 🙏

Reply:
@kelvin-muchiri has this ticket been created?

Reply:
@moshthepitt  Yes